packages/cli/tests/e2e/solution/SSOTabE2ETestWithAADManifestEnabled.tests.ts

```diff
@@ -0,0 +1,89 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import path from "path";
+import {
+  getTestFolder,
+  getUniqueAppName,
+  cleanUp,
+  readContextMultiEnv,
+  setAadManifestIdentifierUris,
+} from "../commonUtils";
+import { CliHelper } from "../../commonlib/cliHelper";
+import {
+  Capability,
+  PluginId,
+  ProjectSettingKey,
+  ResourceToDeploy,
+  TestFilePath,
+} from "../../commonlib/constants";
+import fs from "fs-extra";
+import { expect } from "chai";
+import { AadValidator, BotValidator, FrontendValidator } from "../../commonlib";
+import appStudioLogin from "../../../src/commonlib/appStudioLogin";
+
+describe("SSO Tab with aad manifest enabled", () => {
+  const testFolder = getTestFolder();
+  const appName = getUniqueAppName();
+  const projectPath = path.resolve(testFolder, appName);
+
+  const env = Object.assign({}, process.env);
+  env["TEAMSFX_AAD_MANIFEST"] = "true";
+  env["TEAMSFX_CONFIG_UNIFY"] = "true";
+
+  after(async () => {
+    await cleanUp(appName, projectPath, true, false, false);
+  });
+
+  it("SSO Tab E2E test with aad manifest enabled", async () => {
+    // Arrange
+    await CliHelper.createProjectWithCapability(appName, testFolder, Capability.Tab, env);
+
+    // Assert
+    {
+      const projectSettings = await fs.readJSON(
+        path.join(projectPath, TestFilePath.configFolder, TestFilePath.projectSettingsFileName)
+      );
+      const activeResourcePlugins =
+        projectSettings[ProjectSettingKey.solutionSettings][
+          ProjectSettingKey.activeResourcePlugins
+        ];
+      const capabilities =
+        projectSettings[ProjectSettingKey.solutionSettings][ProjectSettingKey.capabilities];
+      expect(activeResourcePlugins.includes(PluginId.Aad)).to.be.true;
```

Comment:
Add assert for manifest exists and permissions.json not exists?

Reply:
Sure